Propose a README (#46)
chaosinventory/chaosinventory: README.md

# Chaosinventory

> Chaosinventory is a hirachical inventory system mainly focused on the needs of

Reviewer: ```suggestion
Chaosinventory is a hierarchical inventory system focused primarily on the needs of
```

event infrastructure.

> It helps at planning and organizing stuff you have on stock, you rent from others

Reviewer: ```suggestion
It helps at planning and organizing items you have in stock, rent from others
```

or need for a particular event in future.

This is the backend of Chaosinventory. It is more or less a fancy database with
some logic behind it.
We try to run an 'api first' approach, so you can develop your own custom apps,

> frontends and scripts as your needs while still using our backend.

Reviewer: ```suggestion
frontends and scripts to fit your needs while still using our backend.
```
